### DISCUSSION TITLE
Make it an option to have seperate tab bar settings (collapsed/expanded) for seperate windows.

### DISCUSSION THREAD

**Discussion by DipsoNOR** [zen-browser/desktop#2312] [Ideas]
Right now if you expand or collapse the tabbar it changes on all open windows. 

I would want an option for it to only change for the current window. 

Some times have a window with a few tab open, and then have a separate window with lots of tabs and more complex work open where i need to see the tab titles.

**Comment by thesoloist-1** [comment]
Agreed. The one thing stopping me replacing arc with zen. I have multiple monitors with multiple browser windows open. Some windows keep the tab bars collapsed to retain screen real estate and my main window on arc stays open. 

If this can be added I can finally make the leap to completely replace Arc with Zen.

**Comment by originade** [comment]
Just ran into this issue and yeah, this is a must have if I want to have media (Youtube, Twitch, etc) on another monitor. Would love the option to be able to run collapsed in a different window compared to my main window.
